PubNub/Data/Managers/PNSubscriber.m

```diff
@@ -1196,17 +1274,16 @@ - (void)handleLiveFeedEvents:(PNSubscribeStatus *)status {
                 
```

Comment:
@jzucker2 We don't need code above anymore, because data envelop deliver this information for us. So please, remove code starting from _NSArray *listOfActiveChannels = [self allObjects];_ and ending by whole _for_ block which follows it.